Unable to use a TTY - input is not a terminal or the right kind of file
Unable to use a TTY - input is not a terminal or the right kind of file
Unable to use a TTY - input is not a terminal or the right kind of file
Unable to use a TTY - input is not a terminal or the right kind of file
INFO flower 2020-11-08 14:13:57,795 | app.py:55 | Flower server running (insecure, 1 rounds)
INFO:root:Starting logging server...

INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
DEBUG flower 2020-11-08 14:13:58,381 | connection.py:35 | ChannelConnectivity.IDLE
INFO flower 2020-11-08 14:13:58,383 | app.py:32 | Opened (insecure) gRPC connection
DEBUG flower 2020-11-08 14:13:58,389 | connection.py:35 | ChannelConnectivity.READY
DEBUG flower 2020-11-08 14:13:58,431 | connection.py:35 | ChannelConnectivity.IDLE
INFO flower 2020-11-08 14:13:58,432 | app.py:32 | Opened (insecure) gRPC connection
DEBUG flower 2020-11-08 14:13:58,437 | connection.py:35 | ChannelConnectivity.READY
INFO flower 2020-11-08 14:13:58,532 | server.py:80 | [TIME] FL starting
DEBUG flower 2020-11-08 14:13:58,541 | server.py:164 | fit_round: strategy sampled 2 clients (out of 2)
Training 14 epoch(s) w/ 469 mini-batches each

Training 14 epoch(s) w/ 469 mini-batches each3

10.244.1.11 - - [08/Nov/2020 14:13:58] "POST /log HTTP/1.1" 200 -
10.244.1.11 - - [08/Nov/2020 14:13:58] "POST /log HTTP/1.1" 200 -
10.244.1.11 - - [08/Nov/2020 14:13:58] "POST /log HTTP/1.1" 200 -
10.244.4.16 - - [08/Nov/2020 14:13:58] "POST /log HTTP/1.1" 200 -
10.244.4.16 - - [08/Nov/2020 14:13:58] "POST /log HTTP/1.1" 200 -
10.244.4.16 - - [08/Nov/2020 14:13:58] "POST /log HTTP/1.1" 200 -
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
Train Epoch: 0 [29376/30016 (98%)] Loss: 0.1932434
Train Epoch: 1 [1216/30016 (4%)] Loss: 0.086962048
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
Train Epoch: 1 [29376/30016 (98%)] Loss: 0.1361151
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
Train Epoch: 2 [1216/30016 (4%)] Loss: 0.024071187
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
Train Epoch: 2 [29376/30016 (98%)] Loss: 0.0779487
Train Epoch: 3 [1216/30016 (4%)] Loss: 0.007815617
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
Train Epoch: 3 [30000/30016 (100%)] Loss: 0.024658
Train Epoch: 4 [576/30016 (2%)] Loss: 0.017532
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
Train Epoch: 4 [30000/30016 (100%)] Loss: 0.011418
Train Epoch: 5 [576/30016 (2%)] Loss: 0.044428
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
Train Epoch: 5 [29376/30016 (98%)] Loss: 0.1660141
Train Epoch: 6 [1216/30016 (4%)] Loss: 0.017334714
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
Train Epoch: 6 [29376/30016 (98%)] Loss: 0.0669433
Train Epoch: 7 [1216/30016 (4%)] Loss: 0.013738024
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
Train Epoch: 7 [28736/30016 (96%)] Loss: 0.0021419
Train Epoch: 8 [1856/30016 (6%)] Loss: 0.059379460
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
Train Epoch: 8 [28096/30016 (94%)] Loss: 0.0071796
Train Epoch: 9 [2496/30016 (8%)] Loss: 0.078629386
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
Train Epoch: 9 [27456/30016 (91%)] Loss: 0.0370555
Train Epoch: 10 [3136/30016 (10%)] Loss: 0.0162781
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.07
Train Epoch: 10 [26816/30016 (89%)] Loss: 0.0500594/30016 (49%)] Loss: 0.023506
Train Epoch: 11 [3776/30016 (13%)] Loss: 0.01595848
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
Train Epoch: 11 [26176/30016 (87%)] Loss: 0.0174654
Train Epoch: 12 [4416/30016 (15%)] Loss: 0.00416414
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
Train Epoch: 12 [26176/30016 (87%)] Loss: 0.0194989
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
INFO:root:Skipping S3 logfile upload as s3_bucket or s3_key was not provided.
INFO:root:Stopping logging server...

{"client_address": "10.244.1.11:33988", "identifier": "0", "levelname": "DEBUG", "name": "flower", "asctime": "2020-11-08 14:13:58,381", "filename": "connection.py", "lineno": "35", "message": "ChannelConnectivity.IDLE"}
{"client_address": "10.244.1.11:33990", "identifier": "0", "levelname": "INFO", "name": "flower", "asctime": "2020-11-08 14:13:58,383", "filename": "app.py", "lineno": "32", "message": "Opened (insecure) gRPC connection"}
{"client_address": "10.244.1.11:33992", "identifier": "0", "levelname": "DEBUG", "name": "flower", "asctime": "2020-11-08 14:13:58,389", "filename": "connection.py", "lineno": "35", "message": "ChannelConnectivity.READY"}
{"client_address": "10.244.4.16:35974", "identifier": "1", "levelname": "DEBUG", "name": "flower", "asctime": "2020-11-08 14:13:58,431", "filename": "connection.py", "lineno": "35", "message": "ChannelConnectivity.IDLE"}
{"client_address": "10.244.4.16:35976", "identifier": "1", "levelname": "INFO", "name": "flower", "asctime": "2020-11-08 14:13:58,432", "filename": "app.py", "lineno": "32", "message": "Opened (insecure) gRPC connection"}
{"client_address": "10.244.4.16:35978", "identifier": "1", "levelname": "DEBUG", "name": "flower", "asctime": "2020-11-08 14:13:58,437", "filename": "connection.py", "lineno": "35", "message": "ChannelConnectivity.READY"}
TimeoutError raised as no request was received for 1200 seconds.
Train Epoch: 13 [4416/30016 (15%)] Loss: 0.01815693
DEBUG flower 2020-11-08 14:35:38,347 | server.py:176 | fit_round received 2 results and 0 failures
DEBUG flower 2020-11-08 14:35:38,434 | server.py:138 | evaluate: strategy sampled 2 clients
DEBUG flower 2020-11-08 14:35:39,328 | server.py:148 | evaluate received 0 results and 2 failures
INFO flower 2020-11-08 14:35:39,329 | server.py:121 | [TIME] FL finished in 1300.7964435247704
INFO flower 2020-11-08 14:35:39,329 | app.py:59 | app_fit: losses_distributed []
INFO flower 2020-11-08 14:35:39,329 | app.py:60 | app_fit: accuracies_distributed []
INFO flower 2020-11-08 14:35:39,329 | app.py:61 | app_fit: losses_centralized []
INFO flower 2020-11-08 14:35:39,329 | app.py:62 | app_fit: accuracies_centralized []
INFO flower 2020-11-08 14:35:39,337 | server.py:133 | evaluate: no clients sampled, cancel federated evaluation
INFO flower 2020-11-08 14:35:39,337 | app.py:79 | app_evaluate: no evaluation result
command terminated with exit code 137
command terminated with exit code 137